.ci/check.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -ex
+
+diff -u <(echo -n) <(gofmt -d -s .)
```

Comment:
`gofmt` when invoked like this will set the exit code so if `set -e` is set (I'm not sure if putting `-e` in shebang makes it work like that) then it can be simply made:

```suggestion
gofmt -d -s .
```